website/docs/plugins/ray_launcher.md

```diff
@@ -105,3 +105,80 @@ $ python train.py task=1,2,3  -m
 [HYDRA] Stopped AWS cluster. since you've set your provider.cache_stopped_nodes to be True, we are not deleting the cluster.
 ```
 
+In the example app config, we've configured the launcher to download ``*.pt`` files created by the app to local ``download`` dir. You should be able to see a ``download`` dir created in your current working dir.
+
+```commandline
+$ tree -L 1
+.
+├── conf
+├── downloads # Created by example app train.py
+├── model
+├── multirun
+└── train.py
+
+$ tree downloads/
+downloads/
+└── multirun
+    └── 2020-05-18
+        └── 15-17-08
+            ├── 0
+            │   └── checkpoint
+            │       └── checkpoint_1.pt
+            ├── 1
+            │   └── checkpoint
+            │       └── checkpoint_2.pt
+            └── 2
+                └── checkpoint
+                    └── checkpoint_3.pt
+```
+
+
+##### Manage Cluster LifeCycle
+- Delete cluster after job finishes remotely
+```commandline
+python train.py \
+hydra/launcher=ray_aws mode=aws \
+hydra.launcher.params.stop_cluster=True \
+hydra.launcher.params.ray_cluster_cfg.provider.cache_stopped_nodes=False \
+random_seed=1  \
+-m
+```
+
+- Keep cluster running after jobs finishes remotely
+```commandline
+python train.py \
+hydra/launcher=ray_aws mode=aws \
+hydra.launcher.params.stop_cluster=False \
+random_seed=1  \
+-m
+```
+
+- Power off EC2 instances without deletion
+```commandline
+python train.py \
+hydra/launcher=ray_aws mode=aws \
+hydra.launcher.params.stop_cluster=True \
+hydra.launcher.params.ray_cluster_cfg.provider.cache_stopped_nodes=True \
+random_seed=1  \
+-m
+```
+
+##### Connecting to ray cluster
+There will be times when you'd want to connect to the ray cluster. You can use ``get_ray_yaml.py`` script to get a temp yaml file created locally and connect via the ``autoscaler`` cli
+``get_ray_yaml.py`` is a hydra app itself so you can configure it easily like any other hydra app.
+
```

Comment:
I don't like that we need this script.
Let's talk about the use cases its addressing and figure out if we can do without it.

Just one reason it's not clear where will the user of the plugin find it.

Reply:
Created a task to track this in ray launcher sprint.